tiny_gltf_loader.h

```diff
@@ -897,6 +940,41 @@ static bool ParseNumberProperty(double *ret, std::string *err,
   return true;
 }
 
+static bool ParseJSONProperty(std::map<std::string, std::string> *ret, std::string *err,
```

Comment:
I don't get why people would use a pointer and not a reference (I know it's the tinygltf style but still)